README.md

```diff
@@ -1373,7 +1355,7 @@ nockBack('zomboFixture.json', function(nockDone) {
 If your tests are using promises then use `nockBack` like this:
 
 ```
-return nockBack('promisedFixture.json')
+return nockBack('promisedFixture.json', { before: beforeFunc })
```

Comment:
I think we can remove the `, { before: beforeFunc }` options here?

Reply:
whoops forgot about that one